conn.go

```diff
@@ -314,6 +314,7 @@ func (c *Conn) LeaveGroup(group uint32) error {
 type bpfSetter interface {
 	Socket
 	bpf.Setter
+	DetachBPF() error
```

Comment:
Let's go with "RemoveBPF" to more closely match the existing "SetBPF" terminology used in this and other Go packages.

Please update the interface comment to say "setting and removing BPF filters.".